week4:JSweek2/3.js

```diff
@@ -0,0 +1,4 @@
+var x
+console.log("the value of my variable x will be: five");
```

Comment:
`five` or `5`?

Reply:
the homework says "explains in words". I misinterpreted :)